app/routes/_protected.tsx

```diff
@@ -49,7 +47,7 @@ export default function ProtectedRoute() {
   if (shouldRedirectToVerifyEmail) {
     return (
       <Redirect
-        to={preserveParams('/verify-email')}
+        to={{ ...location, pathname: '/verify-email' }}
```

Comment:
shouldn't we set redirectTo param here?

also should we do the same thing for the `if (shouldRedirectToSignup) {` case above?